base_location_nuts/wizard/nuts_import.py

```diff
@@ -136,20 +136,24 @@ def _download_nuts(self, url_base=None, url_path=None, url_params=None):
         url += "&".join([k + "=" + v for k, v in url_params.items()])
         logger.info("Starting to download %s" % url)
         try:
-            res_request = requests.get(url)
+            http = urllib3.PoolManager()
+            res_request = http.request("GET", url)
         except Exception as e:
             raise UserError(
                 _("Got an error when trying to download the file: %s.") % str(e)
-            )
-        if res_request.status_code != requests.codes.ok:
+            ) from e
```

Comment:
Is this change needed?

Reply:
Yes, it is a change due to the good presentation of the error, otherwise it raised a warning.